administrator/language/de-DE/com_cpanel.ini

```diff
@@ -5,38 +5,37 @@
+COM_CPANEL_MSG_ROBOTS_TITLE="Änderungen an der Standard robots.txt-Datei"
 COM_CPANEL_MSG_STATS_COLLECTION_BODY="<p>Seit Joomla! 3.5 sendet ein Statistik-Plugin anonyme Daten zum Joomla!-Projekt. Dabei werden nur die Joomla!- und PHP-Version, der Datenbanktyp und -version und das Serverbetriebssystem gesendet.</p><p>Diese Daten werden für zukünftige Joomla!-Versionen erhoben, um sicherzustellen, dass diese von den neusten Datenbank- und PHP-Funktionen profitieren können, ohne das vorhandene Joomla!-Benutzer Probleme bekommen. Die Notwendigkeit wurde klar, als als Minimum PHP 5.3.10 für Joomla! 3.3 notwendig wurde, da hier die sichere Passwortspeicherung mit Bcrypt eingeführt wurde.</p><p>Im Interesse voller Transparenz und um Erweiterungsentwicklern zu helfen, sind die <a href='https://developer.joomla.org/about/stats.html' target='_blank' rel='noopener noreferrer'>Daten öffentlich einsehbar.</a> Eine API und Diagramme zeigen die verwendeten Joomla!-Versionen, PHP-Versionen und Datenbanktypen.</p><p>Wenn das Senden dieser Daten an das Joomla!-Projekt unerwünscht ist, kann das Plugin „System - Joomla!-Statistikerhebung“ deaktiviert werden.</p>"
 COM_CPANEL_MSG_STATS_COLLECTION_TITLE="Statistikerhebung in Joomla"
 COM_CPANEL_MSG_UPDATEDEFAULTSETTINGS_BODY="<p>Als Teil der Überprüfung des Joomla! Sicherheitsteams (JSST) wurden einige Änderungen an den Standardeinstellungen von Joomla-Installation vorgenommen. Da diese Änderungen nur auf Neuinstallationen angewendet werden, empfehlen wir dringend, diese Änderungen zu überprüfen und die Website entsprechend anzupassen. Die geänderten Einstellungen sind:</p><ul><li>Globale Konfiguration > Textfilter: Die Standard-Benutzergruppe 'Administrator' hat sich von 'Keine Filterung' zu 'Standard Blacklist' geändert</li><li>Benutzer > Passwort senden: Die Option, einem Benutzer sein Passwort im Klartext zu senden, wenn ein Benutzerkonto erstellt wird, ist nun standardmäßig deaktiviert</li><li>Media Manager: Flash-Dateien ('swf' Dateiendung und 'application/x-shockwave-flash' MIME Type) dürfen nicht mehr hochgeladen werden </li><li>Artikel > E-Mail anzeigen: Die Option, ein E-Mail-Symbol in Artikeln anzuzeigen, ist standardmäßig deaktiviert</li></ul>Wir haben eine <a href='https://docs.joomla.org/Special:MyLanguage/J3.x:Joomla_3.8.8_notes_about_the_changed_default_settings'>dedizierte Dokumentationsseite</a> erstellt die diese Änderungen im Detail erklären.</p>"
 COM_CPANEL_MSG_UPDATEDEFAULTSETTINGS_TITLE="Aktualisierte Sicherheitsempfehlungen"
+COM_CPANEL_TITLE_SYSTEM_PANEL="System Panel"
```

Comment:
@joomlagerman/reviewer we need a translation

Reply:
```suggestion
COM_CPANEL_TITLE_SYSTEM_PANEL="Kontrollzentrum"
```

??